libs/designsystem/item/src/item.component.integration.spec.ts

```diff
@@ -36,28 +39,46 @@ describe('ItemComponent with LabelComponent', () => {
           </kirby-label>
         </kirby-item>
         `
-      );
-      ionItem = spectator.queryHost('ion-label');
-      await TestHelper.whenReady(ionItem);
-      labelElements = spectator.queryAll(
-        'ion-item ion-label > :is(h1, h2, h3, h4, h5, h6, p, data)'
-      );
-    });
+        );
+        ionItem = spectator.queryHost('ion-label');
+        await TestHelper.whenReady(ionItem);
+        labelElements = spectator.queryAll(
+          'ion-item ion-label > :is(h1, h2, h3, h4, h5, h6, p, data)'
+        );
+      });
+
+      it('should render general header, data and paragraph elements with correct font-weight', () => {
+        labelElements
+          .filter((e) => !e.attributes.getNamedItem('detail'))
+          .forEach((e) => {
+            expect(e).toHaveComputedStyle({ 'font-weight': fontWeight('bold') });
+          });
+      });
 
-    it('should render general header, data and paragraph elements with correct font-weight', () => {
-      labelElements
-        .filter((e) => !e.attributes.getNamedItem('detail'))
-        .forEach((e) => {
-          expect(e).toHaveComputedStyle({ 'font-weight': fontWeight('bold') });
-        });
+      it('should render detail data and paragraph elements with correct font-weight', () => {
+        labelElements
+          .filter((e) => !!e.attributes.getNamedItem('detail'))
+          .forEach((e) => {
+            expect(e).toHaveComputedStyle({ 'font-weight': fontWeight('normal') });
+          });
+      });
     });
+  });
+
+  describe('when configured with selectable="true" and nested interactive element', () => {
```

Comment:
Consider removing this if you do add the suggestion below
```suggestion
  describe('when configured with selectable="true"', () => {
```